### DISCUSSION TITLE
Testing with LED and ultrasonic components

### DISCUSSION THREAD

**Discussion by guimaraaes** [bxparks/AUnit#98] [General]
Hello everyone,

I'm trying to test a system that activates an LED when an ultrasonic sensor detects an object at a distance greater than or equal to 30cm. My goal is to confirm that the LED is being activated when the ultrasonic sensor measures a distance of 30cm or more. I'm looking for suggestions on specific testing methods or scenarios that are especially useful for verifying the functionality of the system.

If possible, could you provide any recommendations on how to implement this test or share similar experiences that you may have had? I appreciate any help or guidance you can offer.

Thank you!

**Comment by bxparks** [comment]
Hi Sara,
This is not much different than the direct email you sent me. If you are at a stage where you can only ask general questions, perhaps a forum like reddit.com or stackoverflow.com will work better for you. If you have specific questions about the AUnit library, you can come back here.
